app/views/posts/show.html.erb

```diff
@@ -1,2 +1,10 @@
-<h1>Posts</h1>
-<p>Show one post for a given user</p>
+<main>
+<section class="postscontainer">
+<div class="postcomment">
+  <h4><%= @post.title %></h4>
+  <p class="commentcounter">Comments: <%= @post.comments_counter || 0%>, Likes: <%= @post.likes_counter || 0%></p>
+  <p><%= @post.text %></p>
+  </div>
+  <%= render partial: 'comment', locals: {post: @post, comments: @post.comments}%>
+  </section>
+  </main>
```

Comment:
- [ ] On the post show page, Please display the author name and post number as available on the wireframe `post #1 by username`. Your view is missing that

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/175051716-42ccb367-e7b3-4894-bdf1-1e6d94f5ada1.png)